Makefile

```diff
@@ -2,6 +2,8 @@ include config.mk
 include npf.mk
 include tests.mk
 
+PG_dev_NAME = "$(PG_NAME)-dev"
```

Comment:
1: that not related to what the commit explain it does
2: I don't like this variable it's just too much code for something that might never be use